src/components/icon/icon.tsx

```diff
@@ -16,11 +16,21 @@ import classNames from 'classnames';
 
 import { CommonProps, keysOf } from '../common';
 
+import { typeToPathMap } from './icon_map';
 import { icon as empty } from './assets/empty';
 import { enqueueStateChange } from '../../services/react';
 
-import { htmlIdGenerator } from '../../services';
-import { colorToClassMap, isNamedColor, NamedColor } from './named_colors';
+import {
+  htmlIdGenerator,
+  withEuiTheme,
+  WithEuiThemeProps,
+} from '../../services';
+import {
+  COLORS as NAMED_COLORS,
+  isNamedColor,
+  NamedColor,
+} from './named_colors';
```

Comment:
instead of importing, renaming, and re-exporting it, you can instead do:
```suggestion
export { COLORS } from './named_colors';
import { isNamedColor, NamedColor } from './named_colors';
```

Reply:
I went ahead and added this in https://github.com/elastic/eui/pull/5967/commits/dd53f031edb9817cf56036a6e9520b0fbf0cd2dd